test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```

Comment:
- [ ] good work on following the previous reviewer's suggestions. now that you've added the test.md to your .gitignore. remember to delete the `test.md` file from your repository. since this file is being ignored you should remove it from your repository Thanks, 👍. you should be fine after you make that change.
you can simply delete the file or you can untrack the file by running the following commands on your current feature branch: 

```bash
 # this command stops the test.md file from being tracked
git rm -r --cached test.md
git add . 
git commit -m "Remove test.md from being tracked"